### PR TITLE
Do not fail if dir exists, when building meta examples

### PR DESCRIPTION
**Pull request by besser82**

### CODE REVIEW

**Comment by besser82** [comment]
Travis fail is not related…

File: examples/meta/generator/generate.py [hunk 1]
```diff
@@ -88,8 +88,13 @@ def translateExamples(inputDir, outputDir, targetsDir, ctagsFile,
             extension = target["FileExtension"]
 
             # Create directory if it does not exist
-            if not os.path.exists(directory):
-                os.makedirs(directory)
+            try:
+                os.makedirs(name=directory, exist_ok=True)
```
Comment (lisitsyn) [inline examples/meta/generator/generate.py]: Is this python3 only thing?

Comment (besser82) [inline examples/meta/generator/generate.py]: Yes, this has been implemented since Pythoon 3.2

Comment (besser82) [inline examples/meta/generator/generate.py]: In the rebased commit I've changed it to `except TypeError:` to make it clearer.

Comment (besser82) [inline examples/meta/generator/generate.py]: Any objections on merging this?

**Comment by besser82** [comment]
Fail on Travis still not related…  I've built successfully on [fc27](https://koji.fedoraproject.org/koji/taskinfo?taskID=19337489) and [fc26](https://koji.fedoraproject.org/koji/taskinfo?taskID=19327920) using this patch.  Merging…

**Pull request merged**